habitat/robots/manipulator.py

```diff
@@ -70,12 +70,12 @@ def __init__(
 
     def reconfigure(self) -> None:
         """Instantiates the robot the scene. Loads the URDF, sets initial state of parameters, joints, motors, etc..."""
-        # TODO: The current implementation requires users to define all the components of the robot in a single URDF. The future will support loading multiple URDF files.
-        if self.sim_obj is None:
-            ao_mgr = self._sim.get_articulated_object_manager()
-            self.sim_obj = ao_mgr.add_articulated_object_from_urdf(
-                self.urdf_path, fixed_base=self._fixed_base
-            )
+        # TODO: The current implementation requires users to define all the components of the robot in a single URDF.
+        # The future will support loading multiple URDF files.
+        ao_mgr = self._sim.get_articulated_object_manager()
+        self.sim_obj = ao_mgr.add_articulated_object_from_urdf(
```

Comment:
If sim_obj is not None, but is_alive this will add a duplicate robot. There should be a check that self.sim_obj is None or not self.sim_obj.is_alive

Reply:
I will try with this suggestion instead. Thanks @aclegg3

Reply:
should we also check robot_base.py?

Reply:
probably best, although I doubt it will be useful in practice since reconfigure on robot base is always called after the manipulator's reconfigure. Will do !

Reply:
Thank you! :)